src/app/shared/model/audio-version.model.ts

```diff
@@ -11,7 +11,7 @@ export class AudioVersionModel extends BaseModel {
   public id: number;
   public uploads: string = '';
   public label: string;
-  public explicit: string = '';
+  public explicit: string = null;
```

Comment:
I'm not sure i follow why this needs to distinguish between `''` and `null`.  Aren't we dealing with 3 states here - blank, Explicit or Clean?

Generally I've been defaulting/decoding values to blank instead of null.  Since the form fields will generally set it back to blank if you delete/unset it in the UI, and I want that to look like it's "unchanged".